### DISCUSSION TITLE
How to control which switches or lights are randomly turned off and on

### DISCUSSION THREAD

**Discussion by dkossman** [isalkind/homebridge-away-mode#26] [General]
Sorry if I missed this in the description but I'm not clear on how to tell the plugin which lights I want to randomize.  In my case I have many switches and lights that I do not want to randomly turn on or off when I'm away.  

thanks for clarifying!

**Comment by isalkind** [comment]
@dkossman This plugin allows you to create virtual sensors that you use to trigger lights. The plugin does not control the lights. Your are creating named sensors in the plugin that will turn on and off according the configuration you specify. Once you have created the sensor(s) and exposed them to HomeKit, you then use the Home (or other HomeKit client app) to create the automations that will control the lights you want. Ex: When the sensor turns on, your automation should turn on the lights that you want to control. When the sensor turns off, your automation should turn off those same lights. I hope that makes sense. The Configuration section in the README has two fairly detailed examples.

**Reply by dkossman** [reply]
thanks - sorry that I missed this.  will install the plugin to try it out!
